src/main/Main.java

```diff
@@ -1,74 +1,217 @@
+                            System.out.println("Nenhum vídeo encontrado na categoria " + categoriaFiltro);
+                        } else {
+                            for (Video video : videosFiltrados) {
+                                System.out.println("Título: " + video.getTitulo());
+                                System.out.println("Descrição: " + video.getDescricao());
+                                System.out.println("Categoria: " + video.getCategoria());
+                                System.out.println("Duração: " + video.getDuracao());
+                                System.out.println("Data da publicação: " + video.getDataPublicacao());
+                                System.out.println();
+                            }
+                        }
+                    } catch (Exception e) {
+                        System.out.println("Erro: " + e.getMessage());
+                    }
+                    break;
+
+                case 7:
+                    List<Video> videosOrdenados = videoManager.ordenarVideoPorData();
+                    if (videosOrdenados.isEmpty()) {
+                        System.out.println("Nenhum vídeo encontrado");
+                    } else {
+                        for (Video video : videosOrdenados) {
+                            System.out.println("Título: " + video.getTitulo());
+                            System.out.println("Descrição: " + video.getDescricao());
+                            System.out.println("Categoria: " + video.getCategoria());
+                            System.out.println("Duração: " + video.getDuracao());
+                            System.out.println("Data da publicação: " + video.getDataPublicacao());
+                            System.out.println();
+                        }
+                    }
+                    break;
+
+                case 8:
+                    System.out.println("\n=== Relatório de Estatísticas ===");
+                    Map<String, Object> relatorio = videoManager.gerarRelatorioEstatisticas();
+                    System.out.println("Número total de vídeos: " + relatorio.get("totalVideos"));
+                    System.out.println("Duração total dos vídeos: " + relatorio.get("totalDuracao") + " minutos");
+                    System.out.println("Número de vídeos por categoria: ");
+                    Map<String, Integer> videosPorCategoria = (Map<String, Integer>) relatorio.get("videosPorCategoria");
+                    for (Map.Entry<String, Integer> entry : videosPorCategoria.entrySet()) {
+                        System.out.println("- " + entry.getKey() + ": " + entry.getValue());
+                    }
+                    break;
+
+                case 9:
+                    System.out.println("Saindo do sistema...");
+                    break;
+
+                default:
+                    System.out.println("Opção inválida.");
+                    break;
             }
-        }
+        } while (opcao != 9);
```

Comment:
Não faria sentido salvar essa condicional em uma variável lógica?
```suggestion
        } while (menuAberto);
```

Reply:
Implementado!